File: Lexer/LeftCurlyToken.cpp
#include "token.h"
#include <string>
using namespace std;

class LeftCurl : public Token{
	bool Token::equals()
	{

	}

	int Token::hashCode()
	{
		return 5;
	}

	string Token::toString()
	{
		return "{";
	}
};

File: src/main/java/parser/LeftCurlyToken.java
package parser;

public class LeftCurlyToken implements Token{
	public boolean equals(final Object other) {
		return other instanceof LeftCurlyToken;
	}
	
	public int hashCode() {
		return 7;
	}
	
	public String toString() {
		return "LeftCurlyToken";
	}

}
